### DISCUSSION TITLE
block spam mail address with rspamd

### DISCUSSION THREAD

**Discussion by Kemya10** [rspamd/rspamd#4192] [Q&A]
Hello
I am a system and network administrator. I'm having trouble with an rspamd configuration. Basically I would like to block an email address from the rspamd web interface which sends me mass messages. If from the web interface this is not possible, tell me how I could do it

could you please help me

**Comment by kgaughan** [comment]
You probably want to do this in the MTA rather than in rspamd. I don't know what you're using, but in Postfix, you can do this:
```
smtpd_recipient_restrictions = reject_unauth_destination, check_sender_access hash:/etc/postfix/sender_access
```
See http://www.postfix.org/postconf.5.html#smtpd_recipient_restrictions for further information. You might also want to set up the likes of sshguard, which, in spite of its name, can also parse the logs of the major MTAs.
